ext/cl_khr_fp16.asciidoc

```diff
@@ -1491,20 +1491,30 @@ is the infinitely precise result.
 | \<= 2 ulp
 | \<= 2 ulp
 
+// 3 operations from the 2 multiplications and 1 subtraction per component
 | *cross*
-| absolute error tolerance of 'max * max * (3 * HLF_EPSILON)', where max is the maximum input operand value
+| absolute error tolerance of 'max * max * (3 * HLF_EPSILON)' per vector component, where _max_ is the maximum input operand magnitude
 | Implementation-defined
 
 | *degrees*
 | \<= 2 ulp
 | \<= 2 ulp
 
+// 0.5         ULP error in sqrt
```

Comment:
Presumably the equation here was derived from the CTS, but fp16 `sqrt` is "correctly rounded" for FULL profile.

Reply:
So I've been assuming that "correctly rounded" incurs a 0.5 ULP cost based on multiply/add/subtract operations being "correctly rounded" in the spec and allotted this value in other geometric error calculations. This assumption may be wrong though...

For example [single precision normalize](https://github.com/KhronosGroup/OpenCL-CTS/blob/ef832c330c95196e4c8a8de0d95a87f2eb8da967/test_conformance/geometrics/test_geometrics.cpp#L1055) has 2.5 ULP assigned for the 2 ULP error in rsqrt plus the error in a multiply.

Reply:
Yes, that makes sense.